test/upload-test-images.sh

```diff
@@ -16,15 +16,10 @@
 
 set -o errexit
 
-: ${1:?"Pass the directories with the test images as arguments"}
-: ${DOCKER_REPO_OVERRIDE:?"You must set 'DOCKER_REPO_OVERRIDE', see DEVELOPMENT.md"}
+: ${KO_DOCKER_REPO:?"You must set 'KO_DOCKER_REPO', see DEVELOPMENT.md"}
 
-DOCKER_FILES="$(find $@ -name Dockerfile)"
-: ${DOCKER_FILES:?"No subdirectories with Dockerfile files found in $@"}
+IMAGE_DIRS="$(find $(dirname $0)/test_images -depth 1 -type d)"
```

Comment:
`-mindepth 1`?

Reply:
Same comment about `maxdepth` and `${REPO_ROOT_DIR}`.